app/blog/20161005-frontend-build-instructions

```diff
@@ -0,0 +1,80 @@
+---
+title: Welcome to the n.io Front-End Team!
+author:  Trisha Allin
+posted: 20161005
+summary:
+  Everything you need to get your front-end build up and running.
+_options:
+  layout: app/layouts/blog.static.hbs
+---
+
+
+## Install your favorite shell script and text editor!
+Feel free to use whichever terminal shell and text editor you work with best.
+
+Recommendations:
+1. Shell script: zsh/oh-my-zsh
+This shell allows you to theme, has helpful plug-ins and shows you which branch you are on.
+https://github.com/robbyrussell/oh-my-zsh
+2. Text editor: Sublime Text, Atom
+Patrick will probably supply you with sublime… but here’s the link just in case
```

Comment:
You'll probably need to put a new line in between lines 19 and 20 so that 'Patrick' doesn't show on the same line as 'Atom' (markdown is weird)